docs/_data/toc.yml

```diff
@@ -78,19 +78,68 @@
     url: " https://support.shotgunsoftware.com/hc/en-us/articles/360024280254-Reference-Manual"
   - page: pipeline-integrations-reference
     children:
-    - text: core-api
-      url: " https://developer.shotgunsoftware.com/tk-core"
```

Comment:
I think we talked about moving Core API up so it's just below Reference alongside Python and Rest APIs.